definitions/npm/redux-devtools-extension_v2.x.x/flow_v0.104.x-/redux-devtools-extension_v2.x.x.js

```diff
@@ -45,7 +72,7 @@ declare type $npm$ReduxDevtoolsExtension$DevToolsOptions = {
 };
 
 declare function $npm$ReduxDevtoolsExtension$composeWithDevTools<A, B>(ab: A => B): A => B;
-declare function $npm$ReduxDevtoolsExtension$composeWithDevTools(options: $npm$ReduxDevtoolsExtension$DevToolsOptions): compose;
+declare function $npm$ReduxDevtoolsExtension$composeWithDevTools(options: $npm$ReduxDevtoolsExtension$DevToolsOptions): Function;
```

Comment:
The function type here is aliased to `any` so it's not very useful, maybe we can use something more specific or even `(...args: Array<any>) => any` would be better?
ref: https://flow.org/en/docs/types/functions/#toc-function-type

Reply:
I will switch it to `(...args: Array<any>) => any`. I did try to use `$Compose` which is what compose is typed as in redux, but there were lots of errors in the tests.